scalajvm/app/models/data/FeedItem.scala

```diff
@@ -1,6 +1,5 @@
 package models.data
 
-import org.joda.time.DateTime
-import scala.xml._
+import java.time.ZonedDateTime
 
-case class FeedItem(time: DateTime, title: String, description: String, link: String, guid: String)
+case class FeedItem(time: ZonedDateTime, title: String, description: String, link: String, guid: String)
```

Comment:
We were using `time.toString()` in our RSS formatter, so this has broken RSS times.

I'll update it to use proper RFC 822 format (which, while looking very strange, seem to be dictated by [the specification](https://validator.w3.org/feed/docs/rss2.html).